polus-graph-pyramid-builder-plugin/src/main.py

```diff
@@ -596,59 +1207,118 @@ def write_csv(cnames,linear_index,f_info,out_path,out_file):
                         dest='inpDir',
                         type=str,
                         help='Path to input images.',
-                        required=True)
-    parser.add_argument('--outDir',               # Pyramid directory
+                        required=True,
+                        )
+
+    parser.add_argument('--outDir',
                         dest='outDir',
                         type=str,
-                        help='The output directory for the flatfield images.',
-                        required=True)
+                        help='Path to output images.',
+                        required=True
+                        )
     
+    parser.add_argument('--bincount',
+                        dest='bin_count',
+                        type=int,
+                        help='Number of bins',
+                        required=True
+                        )
+
     """ Get the input arguments """
     args = parser.parse_args()
 
     input_path = args.inpDir
     output_path = Path(args.outDir)
+    bincount = args.bin_count
+
+
+    # linear_output_path = Path(args.outDir)
+    # log_output_path = Path(args.outDir)
 
     logger.info('inpDir = {}'.format(input_path))
     logger.info('outDir = {}'.format(output_path))
+    # logger.info('outDirLinear = {}'.format(linear_output_path))
+    # logger.info('outDirLog = {}'.format(log_output_path))
 
     # Get the path to each csv file in the collection
     input_files = [str(f.absolute()) for f in Path(input_path).iterdir() if ''.join(f.suffixes)=='.csv']
 
     for f in input_files:
+
+        # Processes for LINEAR SCALED GRAPHS
         # Set the file path folder
         folder = Path(f)
         folder = folder.name.replace('.csv','')
         logger.info('Processing: {}'.format(folder))
-        
+
         # Load the data
-        logger.info('Loading csv: {}'.format(f))
+        logger.info('Loading LINEAR csv: {}'.format(f))
         data, cnames = load_csv(f)
         column_names = data.columns
-        logger.info('Done loading csv!')
+        logger.info('Done loading LINEAR csv!')
```

Comment:
Rather than inserting `LINEAR` into the middle of each log output, it might be a good idea to do something like the following:
```python
logger.info('LINEAR: message`)
```

This will make the logs easier to read because you don't have to search a sentence to find the LINEAR keyword.

Alternatively, you can generate two loggers. You can create a linear logger by doing the following:
```python
linear_logger = logging.getLogger("main.LINEAR")
linear_logger.setLevel(logging.INFO)
```

Then you don't need to worry about inserting LINEAR into each log entry, and it will look nicer.